.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           command: |
             ccache -z
             # The following packages are currently used in the main pyodide test suite
-            PYODIDE_PACKAGES="micropip,pyparsing,pytz,packaging,kiwisolver" make
+            PYODIDE_PACKAGES="micropip,pyparsing,pytz,packaging,kiwisolver,Jinja2" make
```

Comment:
I think it would be better to use an existing package for tests, for instance pytz, without adding Jinja2 here.

Reply:
pytz's name is all lower case so it doesn't work for this though.